util/devel/grepfuturetests

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
```

Comment:
Given the commonality between these scripts it seems like it'd be nice to have a common script that you can pass a few args to instead of duplicating the code.

Reply:
I do like that idea. Let me work on that